config/secrets.yml.example

```diff
@@ -50,3 +71,13 @@ production:
       signature_public_key: |
         ssm(sso_signature_public_key)
       encryption_private_key: ssm(sso_encryption_private_key)
+  content:
+      abl_url: <%= ENV['OPENSTAX_CONTENT_ABL_URL'] || 'https://raw.githubusercontent.com/openstax/content-manager-approved-books/master/approved-book-list.json' %>
+      archive_path: <%= ENV['OPENSTAX_CONTENT_ARCHIVE_PATH'] %>
+      bucket_name: <%= ENV['OPENSTAX_CONTENT_BUCKET_NAME'] %>
+      domain: <%= ENV['OPENSTAX_CONTENT_DOMAIN'] || 'openstax.org' %>
+      s3_region: <%= ENV['OPENSTAX_CONTENT_S3_REGION'] %>
+      s3_access_key_id: <%= ENV['OPENSTAX_CONTENT_S3_ACCESS_KEY_ID'] %>
+      s3_secret_access_key: <%= ENV['OPENSTAX_CONTENT_S3_SECRET_ACCESS_KEY'] %>
+      rex_release_url: <%= ENV['REX_RELEASE_URL'] || 'https://openstax.org/rex/release.json' %>
```

Comment:
highlights deployments don't... really know what environment they're going to be used in, they know which accounts env they're supposed to use, but that isn't quite the same thing. we're either going to have to add a parameter to pull all the way down through the deployment scripts or deal with it being wrong sometimes in lower environments